packages/flutter_tools/test/android/android_workflow_test.dart

```diff
@@ -39,7 +39,6 @@ void main() {
 
   testUsingContext('licensesAccepted throws if cannot run sdkmanager', () async {
     processManager.succeed = false;
-    MockAndroidSdk.createSdkDirectory();
```

Comment:
Curious: were these probably just copy/pasted from `android_sdk_test.dart`?

Reply:
no idea. It took me a while to convince myself these are no-ops as far as the test is concerned.